spring-integration-cassandra/src/main/resources/org/springframework/integration/config/xml/spring-integration-cassandra-1.0.xsd

```diff
@@ -17,5 +17,139 @@
 		]]></xsd:documentation>
 	</xsd:annotation>
 
-
-</xsd:schema>
+	<xsd:element name="outbound-channel-adapter">
+		<xsd:annotation>
+			<xsd:documentation>
+				Defines cassandra outbound channel adapter that
+				writes the contents of the
+				Message into Cassandra cluster
+			</xsd:documentation>
+		</xsd:annotation>
+		<xsd:complexType>
+			<xsd:complexContent>
+				<xsd:extension base="cassandraAdapterType">
+					<xsd:choice minOccurs="0" maxOccurs="2">
+					<xsd:element name="parameter-expressions" type="queryParameterType" minOccurs="0" maxOccurs="unbounded">
+					<xsd:annotation>
+						<xsd:documentation>
+							TODO
+						</xsd:documentation>
+					</xsd:annotation>
+				    </xsd:element>
+					<xsd:element ref="integration:poller" minOccurs="0" maxOccurs="1"/>
+					<xsd:element name="request-handler-advice-chain" type="integration:handlerAdviceChainType" minOccurs="0" maxOccurs="1" />
+					</xsd:choice>
+				</xsd:extension>
+			</xsd:complexContent>
+		</xsd:complexType>
+	</xsd:element>
+	<xsd:complexType name="cassandraAdapterType">
+		<xsd:annotation>
+			<xsd:documentation>
+				Common configuration for cassandra adapters.
+			</xsd:documentation>
+		</xsd:annotation>
+		<xsd:attributeGroup ref="integration:channelAdapterAttributes" />
+		<xsd:attribute name="cassandra-template" type="xsd:string">
+			<xsd:annotation>
+				<xsd:appinfo>
+					<tool:annotation kind="ref">
+						<xsd:documentation>
+						<![CDATA[
+							Reference to an instance of
+							org.springframework.data.cassandra.core.CassandraOperations]]>
+						</xsd:documentation>
+						<tool:expected-type
+							type="org.springframework.data.cassandra.core.CassandraOperations" />
+					</tool:annotation>
+				</xsd:appinfo>
+			</xsd:annotation>
+		</xsd:attribute>
+		<xsd:attribute name="write-options" type="xsd:string">
+			<xsd:annotation>
+				<xsd:appinfo>
+					<tool:annotation kind="ref">
+						<xsd:documentation><![CDATA[
+							Reference to an instance of
+							org.springframework.cassandra.core.WriteOptions]]>
+						</xsd:documentation>
+						<tool:expected-type
+							type="org.springframework.cassandra.core.WriteOptions" />
+					</tool:annotation>
+				</xsd:appinfo>
+			</xsd:annotation>
+		</xsd:attribute>
+		<xsd:attribute name="output-channel" type="xsd:string">
```

Comment:
This isn't common attribute. And in our case it should be declared in the separate `gateway` element.
See `spring-integration-amqp.xsd` for sample.